### ISSUE TITLE
[template] should a REC allow new features goes to the Standards section or the Normative Specifications section?

### ISSUE THREAD

**Issue by siusin** [w3c/charter-drafts#417]
The [charter assistant](https://www.w3.org/2004/01/pp-impl/charter-assistant?wgid=32061) puts all RECs in a Standards section, to indicate they are in the stable version. 

How about those RECs [allowing new features](https://www.w3.org/2021/Process-20211102/#allow-new-features)? The group still publishes new versions of these specs frequently.

**Comment by plehegar** [comment]
imho, the safest approach here is to always list the latest Call for Exclusion for ALL RECs, whether or not the REC allows new features or not.

**Comment by svgeesus** [comment]
Related:

 - https://github.com/w3c/charter-drafts/issues/385

**Comment by svgeesus** [comment]
The charter assistant currently makes two lists of normative specifications:

1. [Draft and Candidate Standards](https://www.w3.org/groups/wg/webfonts/deliverables/#draft-candidate-standards)
2. [Standards](https://www.w3.org/groups/wg/webfonts/deliverables/#standards)

The [Process calls for](https://www.w3.org/policies/process/#exclusion-draft)

> The title, stable URL, and publication date of the document that was used as the basis for its most recent Exclusion Opportunity as per the W3C Patent Policy [[PATENT-POLICY]](https://www.w3.org/policies/process/#biblio-patent-policy). (labeled “Exclusion Draft”);)

which leads us to the Patent Policy, which loops back to Process by [defining a Patent Review Draft](https://www.w3.org/policies/patent-policy/#patent-review-draft):

> For the purpose of this policy, “Specification” refers to a [W3C technical report](https://w3c.github.io/w3process/#technical-report) published on the [Recommendation Track](https://w3c.github.io/w3process/#rec-track), see [[PROCESS]](https://www.w3.org/policies/patent-policy/#biblio-process). “Patent Review Draft refers to a version of a W3C [Specification](https://www.w3.org/policies/patent-policy/#specification) defined [as such](https://w3c.github.io/w3process/#patent-review-drafts) by the W3C Process [PROCESS], that is published for patent review and exclusion.

Sadly the Process, in the section [The W3C Recommendation Track](https://www.w3.org/policies/process/#rec-track) shows a flow diagram which ends with Rec. No steps past Rec are defined, which makes Patent Review Draft unclear for Recs with proposed or candidate additions.

I recall complaining about this in the past and being told to make a second diagram showing post-Rec stages. I can't now find that discussion, and the related text should in any case be created by (or at least reviewed by) the Process CG.

However Pubrules is at least clear, under [Publishing a Recommendation with proposed additions (new features)](https://www.w3.org/Guide/transitions/?profile=REC&rec=proposed-additions) it states:

> The W3C Communications Team sends the [transition announcement](https://www.w3.org/Guide/transitions/?profile=REC&rec=proposed-additions#trans-annc) to w3c-ac-members@w3.org and chairs@w3.org and on the [W3C home page](https://www.w3.org/). The Call for Exclusions and the Advisory Committee review are started.

So a Rec with proposed additions triggers an Exclusion Opportunity and is thus a Patent Review Draft. So if one has been published, the charter assistant should list the Exclusion Draft.

_It should also not call such a Rec a Draft, but that is a [separate issue](https://github.com/w3c/charter-drafts/issues/627)._

So to my mind, the list of Standards should include any Exclusion Recs that have been published. I don't know if they do, currently.

**Comment by svgeesus** [comment]
Okay, the charter assistant fails to note these. Here is an example:

> [Media Queries Level 3](https://www.w3.org/TR/2024/REC-mediaqueries-3-20240521/)
> [W3C Recommendation](https://www.w3.org/standards/types#REC), 21 May 2024

included a proposed correction, as [announced](https://lists.w3.org/Archives/Member/w3c-css-wg/2024AprJun/0151.html) (Member-only link)

so it was a Patent Review Rec, and the [charter assistant for CSS](https://www.w3.org/groups/wg/css/deliverables/) does not list it.

**Comment by svgeesus** [comment]
@vivienlacourba @deniak it seems that the charter assistant needs to be updated to find these Patent Review Recs and to list them in the Standards section (and not call them drafts).

**Comment by deniak** [comment]
Indeed, the charter assistant wasn't updated after the process started allow REC amendments.
What would be the expectation here? Most of the time, if not all the time, the Patent Review document will be the latest published version we already list. Should we simply add a link to the CfE and mention the exclusion period?

**Comment by svgeesus** [comment]
Related:

 - https://github.com/w3c/process/issues/938

**Comment by svgeesus** [comment]
> Most of the time, if not all the time, the Patent Review document will be the latest published version we already list. 

Agreed.

> Should we simply add a link to the CfE and mention the exclusion period?

I think that would be more consistent with the Draft and Candidate Standards section.

**Comment by plehegar** [comment]
Also related https://github.com/w3c/charter-drafts/pull/651

**Comment by svgeesus** [comment]
> Indeed, the charter assistant wasn't updated after the process started allow REC amendments.

@deniak  Has it now been updated to do that?

**Comment by deniak** [comment]
> > Indeed, the charter assistant wasn't updated after the process started allow REC amendments.
> 
> [@deniak](https://github.com/deniak) Has it now been updated to do that?

@svgeesus, not yet. I'll try to wrap this up this week.

**Comment by deniak** [comment]
@svgeesus @plehegar @siusin I have a pending PR to mention the exclusion draft for the RECs that triggered a CfE.
Here's what the charter assistant would look like for such specs

<img width="694" alt="Image" src="https://github.com/user-attachments/assets/644116af-739d-49fa-b208-dae6f28e82bb" />

Would this be enough or do you expect additional data?

**Comment by svgeesus** [comment]
> Would this be enough or do you expect additional data?

That looks fine to me

**Comment by deniak** [comment]
@svgeesus thanks for the review. The charter assistant has now been updated to include the exclusion draft for these RECs
